imagebuildah/build.go

```diff
@@ -670,6 +670,11 @@ func baseImages(dockerfilenames []string, dockerfilecontents [][]byte, from stri
 							child.Next.Value = from
 							from = ""
 						}
+						if replaceBuildContext, ok := additionalBuildContext[child.Next.Value]; ok {
+							if replaceBuildContext.IsImage {
+								child.Next.Value = replaceBuildContext.Value
+							}
+						}
```

Comment:
Yes giving a warning here that `additional-build-context` is ignored instead of silently ignoring.

Reply:
I have changed this to error since user did this intentionally and it is not supposed to work so behavior which user is expecting and which buildah is giving might differ hence error makes more sense here.